src/linter/block.go

```diff
@@ -1450,7 +1451,11 @@ func (b *BlockWalker) handleSwitch(s *stmt.Switch) bool {
 
 		// allow to omit "break;" in the final statement
 		if idx != len(s.Cases)-1 && bCopy.exitFlags == 0 {
-			b.r.Report(c, LevelInformation, "caseBreak", "Case without break")
+			// allow the fallthrough if appropriate comment is present
+			nextCase := s.Cases[idx+1]
+			if !b.caseHasFallthroughComment(nextCase) {
+				b.r.Report(c, LevelInformation, "caseBreak", "Case without break")
```

Comment:
I suggest that we add some hint here, like this: "Case without break (you can add '// fallthrough' comment to mark that break is not needed here)"

Reply:
Done.